src/components/FabricEditor/fields/FabricCheckbox.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useCallback } from 'react';
+import { Checkbox } from 'antd';
+
+const FabricCheckbox: React.FC<{
+  value: boolean;
+  onChange: (value: boolean) => void;
+  label: string;
+  name: string;
+  type?: string;
+}> = ({ value, onChange, label, name, ...props }) => {
+  const onChangeField = useCallback(
+    (event: any) => {
```

Comment:
[](https://github.com/ant-design/ant-design/blob/master/components/checkbox/Checkbox.tsx ) ?

Reply:
I have no idea how to import it